src/many-abci/src/abci_app.rs

```diff
@@ -76,10 +89,50 @@ impl AbciApp {
             app_name,
             many_url,
             many_client,
+            cache: Arc::new(RwLock::new(())),
```

Comment:
The implementation of cache for `()` is simply to not fail.

Reply:
Ok, so that thing
```
impl RequestCacheBackend for () {
    fn has(&self, _request: &[u8]) -> bool {
        false
    }
    fn put(&mut self, _request: &[u8]) {}
}
```